.NET/Microsoft.Recognizers.Text/Extractors/IExtractor.cs

```diff
@@ -11,6 +11,8 @@ public interface IExtractor
 
     public class ExtractResult : ICloneableType<ExtractResult>
     {
+        public string Issuer { get; set; } = null;
```

Comment:
These attributes should not be added to the base class for ExtractResult, and also not in the Text project.
These are specific of the CC entity type. Perhaps add them in a resolution attribute. Like the case for relative ordinals, or datetimes with its many attributes.